### DISCUSSION TITLE
Triggering console commands from HTTP request

### DISCUSSION THREAD

**Discussion by jamesmacwhite** [craftcms/cloud#47] [Q&A] [answered]
Because there is no native cron functionality currently, we are currently triggering queue jobs via a cron web controller. However, is it possible to trigger a console command by a HTTP request?

**Comment by angrybrad** [comment]
> Because there is no native cron functionality currently,

Very soon!

> However, is it possible to trigger a console command by a HTTP request?

Potentially, but it'll be gross. We're hoping to roll out proper cron job support this week if you can wait a bit.

**Reply by jamesmacwhite** [reply]
Sounds like native cron is very soon, so happy to wait and use queue jobs for now!

**Comment by jasonmccallister** [comment, marked as answer]
CRON support is now available in **Commands** &rarr; **Scheduled Commands**!

More info is available in our KB article, [Scheduled Commands on Craft Cloud](https://craftcms.com/knowledge-base/cloud-cron).

![CleanShot 2024-12-02 at 15 18 42@2x](https://github.com/user-attachments/assets/70ba3285-e9e4-49d8-a2c8-89aedb5fd5ba)

**Reply by jamesmacwhite** [reply]
Thanks for this! Will have a look
